run-directive/action.yml

```diff
@@ -0,0 +1,58 @@
+name: Run Directive
+run-name: test
+
+inputs:
+  repository:
+    description: 'Github repo'
+    required: true
+  hash:
+    description: 'Github commit hash to checkout'
+    required: false
+    default: ''
+  working-directory:
+    description: 'Name of component directory'
+    required: true
+  config:
+    description: 'Config JSON file'
+    required: true
+  stage:
+    description: 'Name of stage'
+    required: true
+  directive:
+    description: 'A directive to run. e.g. npm ci, npm run test'
```

Comment:
Not sure if this is what we were thinking of it doing. Since different actions we might call are called differently, I figured the directive including `npm` or `npm run` would be ideal.

Reply:
I was thinking this should be one of an enum of 'deploy', 'test', or 'teardown'.

Reply:
Sounds good. That was my first thought, but I started second guessing myself and went with this.

Reply:
When you say it should be one of a enum, do you mean we should restrict it to one of these values? I don't think that is something that github actions support. But we could have a step to check the value is one of those three and fail if it's not.